main/glossary/README.md

```diff
@@ -33,19 +33,16 @@ disappear from the total allocation.
 
 ## AmountMath
 The AmountMath library executes the logic of how [amounts](#amounts) are changed when digital assets are merged, separated,
-or otherwise manipulated. For example, a deposit of 2 bucks into a purse that already has 3 bucks
-gives a new balance of 5 bucks. But, a deposit of a non-fungible theater ticket into a purse that already holds
-five tickets isn't done by numeric addition. Instead, you have to combine two arrays, containing either
-strings or objects/records.
-
-`AmountMath` has a single set of polymorphic
-methods of two different asset kinds to deal with [fungible](#fungible) assets (values are natural numbers) and
-[non-fungible](#non-fungible) assets (values are an array or object). The two `AssetKinds` are
-- `AssetKind.NAT`: Used with fungible assets. Amount values are natural numbers (non-negative integers). Default value.
-- `AssetKind.SET`: Used with non-fungible assets. Amount values are
-  arrays of strings, numbers, objects, or other comparable values.
-  Values should never include promises (they aren't comparable), or
-  payments, purses, and anything else that can't be shared freely.
+or otherwise manipulated. For example, a deposit of 2 [Quatloos](#quatloos) into a purse that already has 3 Quatloos
+gives a new balance of 5 Quatloos. But a deposit of a non-fungible theater ticket into a purse that already holds
+five tickets is performed by combining arrays rather than by arithmetic.
+
+`AmountMath` has a single set of polymorphic methods that deal with two different asset kinds:
```

Comment:
```suggestion
`AmountMath` has a single set of polymorphic methods that deal with different asset kinds:
```
Add sections below on AssetKind.COPY_SET and AssetKind.COPY_BAG . Mention that AssetKind.SET is recently deprecated in favor of AssetKind.COPY_SET but still in wide use. AssetKind.COPY_BAG is for semi-fungible.